### ISSUE TITLE
Pysa timestamp format change

### ISSUE THREAD

**Issue by captainGeech42** [captainGeech42/ransomwatch#50]
```
Traceback (most recent call last):
  File "/app/ransomwatch.py", line 66, in main
    s.scrape_victims()
  File "/app/sites/pysa.py", line 38, in scrape_victims
    published_dt = datetime.strptime(
  File "/usr/local/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '22/03/21' does not match format '%m/%d/%y'
```

**Comment by biligonzales** [comment]
Hello @captainGeech42 
I have started working on a as-clean-as-possible solution about this issue on my fork. 
I can PR as soon as it's fully tested :)

**Comment by captainGeech42** [comment]
Awesome, thanks!

**Comment by biligonzales** [comment]
A bit of testing has been done, however the platform went offline since them and I couldn't test more. 
I think the patch is quite good though. Could be enhanced later when the platform is back too.

**Comment by captainGeech42** [comment]
Merged in #54
